docs/source/install.rst

```diff
@@ -45,14 +45,17 @@ the binaries and images.
 
   curl -sSL https://bit.ly/2ysbOFE | bash -s
 
+.. note:: You can use ``-h`` option at the end to list the options available for the script.
+          The command in that case becomes ``curl -sSL https://bit.ly/2ysbOFE | bash -s -- -h``
+
 .. note:: If you want a specific release, pass a version identifier for Fabric and Fabric-CA docker images.
           The command below demonstrates how to download the latest production releases -
           **Fabric v2.2.0** and **Fabric CA v1.4.8**
 
 .. code:: bash
 
-  curl -sSL https://bit.ly/2ysbOFE | bash -s -- <fabric_version> <fabric-ca_version>
-  curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.0 1.4.8
+  curl -sSL https://bit.ly/2ysbOFE | bash -s -- <fabric_version> <fabric-ca_version> <script_option>
+  curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.0 1.4.8 -s
```

Comment:
Adding the `-s` flag here actually isn't appropriate, while it make sense now, because the current master of Fabric-Samples is what is used for the `2.2.0` images today, once we cut the `2.2.0` tag in fabric samples the command will no longer be appropriate. I think these two lines should be reverted in favor of my comments above.

Today the preceding commands only have you clone the repo, they don't have you checkout a specific tag. So adding the `-s` flag would prevent actually checking out the correct tag in the coming weeks.